app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
```

Comment:
Consider checking if the `command` string contains exactly three elements and starts with 'cp'. This is important to ensure the command is valid as per the checklist requirements.